config.sh

```diff
@@ -1 +1,2 @@
 export VFDEPS_NAME=vfdeps-7c315b0
+export VF_LLVM_CLANG_BUILD_VERSION=a5d9179
```

Comment:
This commit is no longer in any branch of the repo? Did you force-push?

Reply:
Indeed. I didn't realise that I was working on the repo's main branch and it contained a lot of WIP commits. I squashed and forced pushed to clean it. Do you want me to rerun the workflow so the artifact's version points to the latest commit?

Reply:
Yes, please. (And while you're at it, perhaps re-introduce a dash between `vf-llvm-clang-build` and the version?) And check that the tag associated with the GitHub release corresponds to the commit that was built.